manifests/install.pp

```diff
@@ -60,9 +60,15 @@
     }
   }
 
+  if $::osfamily == 'Darwin' {
+    $path = '/var/root'
```

Comment:
Given `$path` is a banned keyword now I think, consider something like `$root_npmrc_path = '/var/root/.npmrc'` instead

Reply:
Oh dear, that's going to cause some fun for me (just starting to move things from puppet 3 to 4 and must have missed this one!)